src/DataAPI.jl

```diff
@@ -73,6 +73,28 @@ definition.
 """
 function describe end
 
+"""
+    levels(x)
+
+Return a vector of unique values which occur or could occur in collection `x`,
+omitting `missing` even if present. Values are returned in the preferred order
+for the collection, with the result of [`sort`](@ref) as a default.
```

Comment:
maybe add that `sort` happens only if the elements of the collection are sortable and that decision to try sorting is based on `eltype` of a collection and not the type of values?

E.g. `levels(Any[3,2,1])` returns `[3,2,1]`. Maybe this should also be added to tests?

Reply:
Good catch. Let's drop the type check, that will make things more reliable.

Reply:
agreed